HIPs/incubation/web3j.md

```diff
@@ -0,0 +1,204 @@
+
+Web3j is a successful open source project and a key piece of open source infrastructure for the Ethereum ecosystem. However, as the projects creator I want to ensure that Web3j ultimately  resides within a foundation that can ensure that Web3j thrives for the long term without being directly tied to Web3 Labs.
+
+With the Hyperledger Foundations role as the leading ecosystem for enterprise-grade blockchain technology, I believe there is no better place for Web3j to reside long term.
+
+# Dependent Projects
+
+Web3j requires an Ethereum compatible network to run. It works natively with any Ethereum node software that implements the execution client [JSON-RPC specification](https://ethereum.github.io/execution-apis/api-documentation/). 
+
+This includes Ethereum clients such as Hyperledger Besu, Quorum, and Geth, as well as appchain and layer two clients such as the OP Stack, Linea and Polygon ZK-EVM.
+
+As both Web3j and Hyperledger Besu run on the JVM, there are a number of existing dependencies between the two projects.
+
+Hyperledger Besu makes use of Web3j in a [number of areas](https://github.com/search?q=repo%3Ahyperledger%2Fbesu%20web3j&type=code), including using its Ethereum ABI type implementations and in its acceptance tests. Web3j makes use of Hyperledger Besu's EVM implementation within its unit testing framework Web3j Unit. This facilitates unit testing of Solidity smart contracts within Web3j projects.
+
+# Motivation
+
+When developers wish to integrate with Ethereum based infrastructure, they are typically trying to accomplish one of the following:
+
+1. Query a blockchain node to obtain information about on-chain activities
+2. Deploy smart contracts
+3. Transact with or query data from deployed smart contracts
+4. Create wallets
+5. Send Ether from one wallet to another
+
+Prior to the creation of Web3j, there was no straightforward way to accomplish any of the above on the JVM. Developers would need to develop their own code to provide the following functionality:
+
+- Implement the JSON-RPC protocol for communicating with Ethereum nodes
+- Perform type conversion for deploying, transacting with and reading data from smart contracts
+- Working with Ethereum's Secp256k1 elliptic curve for signing transactions
+
+Web3j was created originally with the goal of addressing these needs for users. However, over time its scope grew to address some of the challenges with the overall development workflow when working with smart contracts on the JVM:
+
+- Scaffolding Solidity projects
+- Providing an integrated developer experience
+- A lack of unit and integration testing capabilities
+
+The specific components of Web3j and how it addresses these items are addressed in subsequent sections below.
+
+For further historical context on Web3j, you can read the [following post I wrote](https://web3perspectives.com/p/building-open-source-software-web3j) on the creation of the project earlier the year.
+
+# Status
+
+Proposal
```

Comment:
```suggestion
Incubation
```

Reply:
Specify which state you are looking for this project to come into within Hyperledger